packages/core/src/event-bus/events/customer-address-event.ts

```diff
@@ -1,21 +1,37 @@
-import { RequestContext } from '../../api/common/request-context';
-import { Address } from '../../entity/address/address.entity';
-import { VendureEvent } from '../vendure-event';
+import { CreateAddressInput, UpdateAddressInput } from '@vendure/common/lib/generated-types';
```

Comment:
@michaelbromley Why am I not able to import this? (I get an error from the check-imports script)

Reply:
This should not raise an error - the check-imports script only complains if it has been imported from `@vendure/common/src/...` rather than `lib`.